statsmodels/genmod/families/family.py

```diff
@@ -431,6 +432,8 @@ class Gaussian(Family):
     links = [L.log, L.identity, L.inverse_power]
     variance = V.constant
 
+    # Gaussian family does not have safe_links since all links are safe
```

Comment:
you can define `safe_links = link`s i.e. as copy of links (so it's defined for all our families)
I think using a list is better than using a tuple so we or users can monkey patch it if necessary.
`family.safe_links.append(mylink)`